guess/src/main/java/ee/eek/guess/handlers/DataHandler.java

```diff
@@ -0,0 +1,39 @@
+package ee.eek.guess.handlers;
+
+import java.util.Map;
+
+import ee.eek.guess.data.PlayerData;
+
+import java.util.HashMap;
+public class DataHandler {
```

Comment:
Nice abstraction over HashMap functionality. 

Just a small recommendation. You can make DataHandler more specific For eg. GameDataHandler/GameRepository. 
Then you can make public only those methods you need for your game